amd_openvx_extensions/amd_rpp/source/SequenceRearrange.cpp

```diff
@@ -0,0 +1,235 @@
+/*
+SequenceRearrangeright (c) 2019 - 2020 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a SequenceRearrange
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, SequenceRearrange, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above SequenceRearrangeright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR SequenceRearrangeRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "internal_publishKernels.h"
+
+struct SequenceRearrangeLocalData {
+
+#if ENABLE_OPENCL
+    RPPCommonHandle handle;
+#endif
+    RppiSize dimensions;
+    RppPtr_t pSrc;
+    RppPtr_t pDst;
+    Rpp32u device_type;
+    vx_uint32 new_sequence_length;
+    vx_uint32 sequence_length;
+    vx_uint32 sequence_count;
+    vx_uint32* new_order;
+
```

Comment:
remove empty lines and make #elif for HIP backend

Reply:
We have not added the GPU support for the SequenceRearrange Node we shall add it.